plugin/src/main/java/net/dzikoysk/funnyguilds/feature/notification/bossbar/provider/BossBarProviderImpl.java

```diff
@@ -21,7 +22,7 @@ public BossBarProviderImpl(User user) {
 
     @Override
     public void sendNotification(String text, BossBarOptions options, int timeout) {
-        user.getPlayer().peek(player -> {
+        Option.of(Bukkit.getPlayer(this.user.getUUID())).peek(player -> {
```

Comment:
Może warto rozważyć jakąś funkcję w stylu `BukkitUtils.getPlayer(user) -> Option<Player>` skoro ta metoda wyleciała z Usera, bo tak to wszedzie trzeba to opakować i wyciągać uuid teraz

Reply:
Będziemy w takim razie używać tego naszego FunnyServer

Reply:
moja dusza może spać spokojnie